src/table/io/FitsSchemaInputMapper.cc

```diff
@@ -708,9 +712,21 @@ bool endswith(std::string const &s, std::string const &suffix) {
     return s.size() >= suffix.size() && s.compare(s.size() - suffix.size(), suffix.size(), suffix) == 0;
 }
 
-// Replace the last n characters of a string with a new suffix string, returning the result as a new string
-std::string replaceSuffix(std::string const &s, std::size_t n, std::string const &suffix) {
-    return s.substr(0, s.size() - n) + suffix;
+bool hasInstFluxUnits(FitsSchemaItem const & item) {
+    // helper lambda to make reading the real logic easier
+    auto includes = [](std::string const & s, char const * target) {
+        return s.find(target) != std::string::npos;
+    };
+    if (!includes(item.ttype, "flux")) return false;
+    // transform units to lowercase.
+    std::string units(item.tunit);
+    std::transform(units.begin(), units.end(), units.begin(), [](char c) { return std::tolower(c); } );
+    return includes(units, "count") || includes(units, "dn") || includes (units, "adu");
```

Comment:
Oh joy: we use all of these units? Have we standardized this yet, or do we need an RFC for it?

Reply:
We certainly don't anymore.  I'm pretty sure that at one point we used `dn`, but that may predate `afw::table`, and I'm not certain we ever used `adu`.  I was just trying to be thorough.